src/plugins/console/public/lib/mappings/__tests__/mapping.test.ts

```diff
@@ -256,10 +257,37 @@ describe('Mappings', () => {
       'test_index2',
     ]);
     expect(mappings.getIndices(false).sort()).toEqual(['test_index1', 'test_index2']);
-    expect(mappings.expandAliases(['alias1', 'test_index2']).sort()).toEqual([
+    expect((mappings.expandAliases(['alias1', 'test_index2']) as string[]).sort()).toEqual([
       'test_index1',
       'test_index2',
     ]);
     expect(mappings.expandAliases('alias2')).toEqual('test_index2');
   });
+
+  test('Multi types', function () {
```

Comment:
Nit: I think there are also some missing tests on other functions like `retrieveAutoCompleteInfo`. Not sure if you could help to add some unit tests here.